Sources/Workspace/Workspace.swift

```diff
@@ -281,6 +285,7 @@ public class Workspace {
         resolverUpdateEnabled: Bool? = .none,
         resolverPrefetchingEnabled: Bool? = .none,
         sharedRepositoriesCacheEnabled: Bool? = .none,
+        strictFingerprintChecking: Bool? = .none,
```

Comment:
`resolverFingerprintChecking[Mode]` as this is a resolver setting. we should probably (not in this PR) group all the resolver settings into a struct

Reply:
https://github.com/apple/swift-package-manager/pull/3890/commits/578ba66a5f4a6e4103b9a8e82cbd73c6d212457b